theme/_provisional.scss

```diff
@@ -0,0 +1,30 @@
+.provisional{
+	.bg-darker {
+		background-color:#000;
+	}
+
+	.bg-darker.well.header-rwd h1 {
+		border-bottom-color:#6f6f6f;
+	}
+
+	.bg-gctheme {
+		background-color:#355688;
+	}
+
+	.bg-gctheme.well.header-rwd {
+		background-color: #26374a;
+	}
+
+	.bg-img-hdng {
```

Comment:
This one should not have been brought over here. It is for a different provisional feature.